agent-inject/agent/agent.go

```diff
@@ -4,33 +4,33 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
-	"strconv"
-	"strings"
-
 	"github.com/mattbaird/jsonpatch"
 	corev1 "k8s.io/api/core/v1"
+	"strconv"
+	"strings"
 )
 
 // TODO swap out 'github.com/mattbaird/jsonpatch' for 'github.com/evanphx/json-patch'
 
 const (
-	DefaultVaultImage                       = "hashicorp/vault:1.8.0"
-	DefaultVaultAuthType                    = "kubernetes"
-	DefaultVaultAuthPath                    = "auth/kubernetes"
-	DefaultAgentRunAsUser                   = 100
-	DefaultAgentRunAsGroup                  = 1000
-	DefaultAgentRunAsSameUser               = false
-	DefaultAgentAllowPrivilegeEscalation    = false
-	DefaultAgentDropCapabilities            = "ALL"
-	DefaultAgentSetSecurityContext          = true
-	DefaultAgentReadOnlyRoot                = true
-	DefaultAgentCacheEnable                 = "false"
-	DefaultAgentCacheUseAutoAuthToken       = "true"
-	DefaultAgentCacheListenerPort           = "8200"
-	DefaultAgentCacheExitOnErr              = false
-	DefaultAgentUseLeaderElector            = false
-	DefaultAgentInjectToken                 = false
-	DefaultTemplateConfigExitOnRetryFailure = true
+	DefaultVaultImage                               = "hashicorp/vault:1.8.0"
+	DefaultVaultAuthType                            = "kubernetes"
+	DefaultVaultAuthPath                            = "auth/kubernetes"
+	DefaultAgentRunAsUser                           = 100
+	DefaultAgentRunAsGroup                          = 1000
+	DefaultAgentRunAsSameUser                       = false
+	DefaultAgentAllowPrivilegeEscalation            = false
+	DefaultAgentDropCapabilities                    = "ALL"
+	DefaultAgentSetSecurityContext                  = true
+	DefaultAgentReadOnlyRoot                        = true
+	DefaultAgentCacheEnable                         = "false"
+	DefaultAgentCacheUseAutoAuthToken               = "true"
+	DefaultAgentCacheListenerPort                   = "8200"
+	DefaultAgentCacheExitOnErr                      = false
+	DefaultAgentUseLeaderElector                    = false
+	DefaultAgentInjectToken                         = false
+	DefaultTemplateConfigExitOnRetryFailure         = true
+	DefaultTemplateConfigStaticSecretRenderInterval = "5m"
```

Comment:
If the user hasn't specified a render interval, I'm thinking we should just not set it and let the default on the vault agent side be the default.
```suggestion
```